examples/http-default-resource/tests/http_default_resource_test.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/http;
+import ballerina/test;
+
+@test:Config {}
+function testFunc() returns @tainted error? {
+
+    http:Client httpEndpoint = check new("http://localhost:9090");
+
+    json payload = {"method":"GET","path":["foo","bar"]};
+    var response = httpEndpoint->get("/foo/bar", targetType = json);
+    if (response is json) {
+        test:assertEquals(response, payload);
+    } else {
+        test:assertFail(msg = "Failed to call the endpoint");
+    }
+
+    string value = "Specific resource is invoked";
```

Comment:
Seems `value` is never reused. Shall we have `test:assertEquals(response, "Specific resource is invoked");` instead.